.soliumrc.json

```diff
@@ -13,7 +13,6 @@
     "security/no-low-level-calls": 0,
     "security/no-inline-assembly": 0,
     "security/no-call-value": 0,
-    "security/no-block-members": 0,
-    "security/enforce-explicit-visibility": 0
+    "security/no-block-members": 0
```

Comment:
Style: We should leave in settings that are default so it is explicit and doesn't change in the future if defaults change. Some goes for the other three missing settings, which I think should be included with an explicit sitting.

This allows the engineer to easily here litter settings without needing to research documentation on how to do so, and also makes it easy to reference what settings we are using.